cilium/cmd/kernel_check.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+
+	"github.com/cilium/cilium/pkg/datapath/linux/probes"
+	"github.com/spf13/cobra"
+)
+
+var kernelCheckCmd = &cobra.Command{
+	Use:   "kernel-check",
+	Short: "Checks whether machine is supporting required kernel capabilities",
+	Run: func(cmd *cobra.Command, args []string) {
+		kernelCheck()
+	},
+}
+
+func kernelCheck() {
+	probeManager := probes.NewProbeManager()
+	w := tabwriter.NewWriter(os.Stdout, 30, 8, 0, '\t', 0)
+
+	requiredParams := probeManager.GetRequiredConfig()
+	fmt.Fprintf(w, "\n %s\t%s\t%s\t", "FEATURE:", "SUPPORTED:", "REQUIRED:")
+	for k, v := range requiredParams {
+		fmt.Fprintf(w, "\n %s\t %t\t true\n", k, v)
+		if v == true {
+			fmt.Fprintf(w, "\n %s\t %t\t true\n", k, v)
+		}
```

Comment:
Isn't that printing twice the same thing if `v == true`?

Reply:
of course, fixed